### DISCUSSION TITLE
Heltec Vision Master E290 - DEPG0290BNS800F6_V2.1 Support?

### DISCUSSION THREAD

**Discussion by SamLue** [olikraus/u8g2#2547] [General]
In my Project I started testing with an SSD1306. But now I'm tring to implement an E-Ink Display.
The Board is the Heltec Vision Master E290 with DEPG0290BNS800F6_V2.1 as Display.

Is it possible to use this display with the u8g2 (or u8x8)?
I already have tried the IL3820 V2_296X128 and IL3820 296X128 construktor. But without success.
It would be realy nice to use the same lib for all displays.

Beste regards,
Sam

**Comment by SamLue** [comment]
I found the [datasheet](https://resource.heltec.cn/download/HT-VME290/DEPG0290BNS800F6_V2.1.pdf) of the display.
DRIVER IC: SSD1680Z8
I can't find the SSD1680Z8 in the [List of supported displays](https://github.com/olikraus/u8g2/wiki/u8g2setupcpp).

**Reply by olikraus** [reply]
I tried to add support for e-ink displays long time back, but they actually break the u8g2 architecture. More over there are better libs, for example this one: https://github.com/ZinggJM/GxEPD2

In fact the SSD1680 seems to be supported by  https://github.com/ZinggJM/ e-ink lib.
